_plugins/report_data_generator.rb

```diff
@@ -0,0 +1,53 @@
+require 'json'
+require 'csv'
```

Comment:
Why CSV? Seems like we should stick to what we're already using instead of introducing more technologies into the mix.